### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]:Name property is empty for button in "Push Image dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1409]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. Navigate to and select Project ->Azure->Push Image.
4. In "push Image" dialog ,tab to button beside  docker file edit and observe the name property using inspect.

**Actual Result:**
Name property is empty for the button 

**Narrator behaviour:**
Tab
button

**NVDA behaviour:**
button

**Expected Result:**
Name property should be simple and meaningful.
Name property for the button should be "Browse".

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![push image-button](https://user-images.githubusercontent.com/22956130/35685732-7c88798c-0790-11e8-9c50-c2334d6c577b.jpg)
![push image-button-nvda](https://user-images.githubusercontent.com/22956130/35685735-7fa88d0a-0790-11e8-89a1-e142daff8e1e.jpg)

**Comment by Eskibear** [comment]
change the wording to "Browse"

**Comment by Eskibear** [comment]
fixed in PR #1590

**Comment by WiproPSAccTest** [comment]
Verified the issue in V.3.6.0 .Issue got fixed.
Please add closed tag.
![1409-fixed](https://user-images.githubusercontent.com/22956130/39760754-d2d8be1a-52f3-11e8-9c92-0bc07d327dd0.jpg)

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749878
